playbooks/roles/nginx/tasks/main.yaml

```diff
@@ -0,0 +1,8 @@
+- name: copy helm yaml template for nginx
+  template:
+    src: nginx-config.yaml
+    dest: /tmp/nginx-config.yaml
+
+- name: install nginx
+  command: >
+    helm install --name nginx-ingress --namespace=nginx-ingress stable/nginx-ingress --values /tmp/nginx-config.yml
```

Comment:
When installing helm charts with ansible we have usually used the `upgrade --install` command since otherwise Ansible play fails if you run playbook a second time and chart is already installed. E.g. from our Luigi ansible role:
```
    helm upgrade --install
      --values ~/.kubernetes-yaml/luigi/helm-values.yaml
      --version "{{ luigi_chart_version }}"
      "luigi-{{ luigi_chart_version }}"
      phenomenal-helm/luigi
```

Reply:
We could also consider a pinned version instead of "stable"?

Reply:
fixed.

Reply:
> We could also consider a pinned specific version instead of "stable"?

No, stable is the helm repo that the template is pulled from.